manifests/certonly.pp

```diff
@@ -106,27 +118,35 @@
     $command_end = join(['',] + $additional_args, ' ')
   }
 
-  $command = "${command_start}${command_domains}${command_end}"
-  $live_path_domain = regsubst($domains[0], '^\*\.', '')
-  $live_path = "${config_dir}/live/${live_path_domain}/cert.pem"
+  # certbot uses --cert-name to generate the file path
+  $live_path_certname = regsubst($title, '^\*\.', '')
```

Comment:
Is it related to this PR ? Looks an other issue.

Reply:
To be fair that is a pre-existing issue, but I needed to fix it to properly test the changes on my real-world setup regardless.

Reply:
It will be hard to find this bugfix in the middle of this PR that is not related with. It will not be listed in the CHANGELOG.md. And so, we should create a dedicated PR about this.

Can you explain the problem about path in an issue, and create a PR ? We will merge the PR about path issue, then rebase this #175, and merge it.

Reply:
Created PR #180 to fix the path issue.

Reply:
#180 is now merged. You can rebase #177